aesara/tensor/random/basic.py

```diff
@@ -2086,6 +2086,20 @@ def __call__(self, x, **kwargs):
 permutation = PermutationRV()
 
 
+class RandomRV(RandomVariable):
+    name = "random"
+    dtype = "floatX"
+    ndim_supp = 0
+    ndims_params = [0, 0]
+    _print_name = ("random", "\\operatorname{U}")
+
+    def __call__(self, size=None, **kwargs):
+        return super().__call__(size=size, **kwargs)
+
+
+random = RandomRV()
```

Comment:
In this case, we need to reuse the existing `UniformRV` `Op`.  Something like `random = uniform` might work.

Reply:
Fixed, Thanks.